packages/ovee/src/reactive/makeComponentReactive.ts

```diff
@@ -0,0 +1,13 @@
+import { WithReactiveProxy } from 'src/core/types';
+
+import { ReactiveProxy } from './ReactiveProxy';
+
+export function makeComponentReactive(instance: WithReactiveProxy): ReactiveProxy {
+	// might be replaced with more sophisticated implementation
+	// using WeakMap, but let's keep it dead simple for now
+	if (!instance.__reactiveProxy) {
```

Comment:
I'd prefer to use a Symbol instead of `__reactiveProxy, as it feels to fragile and exposed to me

Reply:
sure, we can move it to `protectedFields`

Reply:
done